report/hipchat.go

```diff
@@ -0,0 +1,130 @@
+package report
+
+import (
+	"fmt"
+	"github.com/future-architect/vuls/config"
+	"github.com/future-architect/vuls/models"
+	"github.com/k0kubun/pp"
+	"net/http"
+	"net/url"
+	"strings"
+	"strconv"
+)
+
+// HipChatWriter send report to HipChat
+type HipChatWriter struct{}
+
+func (w HipChatWriter) Write(rs ...models.ScanResult) (err error) {
+	conf := config.Conf.HipChat
```

Comment:
I think these local variables (conf, room and token) have no meaning.
Use config.Conf.conf, room, token directory.